themes/default/content/docs/pulumi-cloud/esc/reference.md

```diff
@@ -161,6 +161,14 @@ values:
           app-secret:
             path: app-secret
 
+  # Pulumi Stacks Provider example
+  app:
+    fn::open::pulumi-stacks:
+      stacks:
+        k8-cluster:
+          stack: k8-cluster-1/dev
+    kubeconfig: {'fn::toJSON': "${app.k8-cluster.kubeconfig}"}
```

Comment:
```suggestion
    kubeconfig: 
      fn::toJSON: ${app.k8-cluster.kubeconfig}
```

Reply:
@arunkumar611 Did you mean to apply these suggestions before merging?